cmd/app/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"context"
+	"embed"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/DropsWeb/todo-go/internal/database/postgres"
+	"github.com/DropsWeb/todo-go/internal/services"
+	"github.com/DropsWeb/todo-go/pkg/swagger"
+	"github.com/DropsWeb/todo-go/pkg/swagger/ops"
+	"github.com/pressly/goose/v3"
+
+	"github.com/go-openapi/loads"
+	"github.com/jackc/pgx/v5/pgxpool"
+	_ "github.com/jackc/pgx/v5/stdlib"
+	"github.com/rs/cors"
+	"github.com/spf13/viper"
+)
+
+var embedMigrations embed.FS
+
+func main() {
+	viper.SetConfigFile("../../.env")
+	viper.ReadInConfig()
+
+	connectData := fmt.Sprintf("user=%s password=%s host=%s port=%s dbname=%s",
+		viper.Get("POSTGRES_USER"),
+		viper.Get("POSTGRES_PASSWORD"),
+		viper.Get("POSTGRES_HOST"),
+		viper.Get("POSTGRES_PORT"),
+		viper.Get("POSTGRES_DBNAME"),
+	)
+
+	connectDataMigrate := connectData + " sslmode=disable"
```

Comment:
Можно сразу указать `sslmode`
```suggestion
	connectData := fmt.Sprintf("user=%s password=%s host=%s port=%s dbname=%s sslmode=disable",
		viper.Get("POSTGRES_USER"),
		viper.Get("POSTGRES_PASSWORD"),
		viper.Get("POSTGRES_HOST"),
		viper.Get("POSTGRES_PORT"),
		viper.Get("POSTGRES_DBNAME"),
	)
```